starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -33,5 +35,14 @@ public InventoryController(InventoryDAO inventoryDAO) {
   public List<Inventory> findInventories() {
     return this.inventoryDAO.findAll();
   }
```

Comment:
If you are going to leave off the class level `@RequestMapping` above, you need to fix the `@GetMapping` here to include a path in it. Otherwise there really is no way to call this method.